doc/api/perf_hooks.md

```diff
@@ -201,6 +201,25 @@ The value may be one of:
 * `perf_hooks.constants.NODE_PERFORMANCE_GC_INCREMENTAL`
 * `perf_hooks.constants.NODE_PERFORMANCE_GC_WEAKCB`
 
+### performanceEntry.flags
+<!-- YAML
+added: REPLACEME
+-->
+
+* {number}
+
+When `performanceEntry.entryType` is equal to `'gc'`, the `performance.flags`
+property contain additional information about garbage collection operation.
+The value may be one of:
```

Comment:
If we are exposing this we at least need a note similar to the one  in https://nodejs.org/api/v8.html#v8_v8_getheapspacestatistics

> Neither the ordering of heap spaces, nor the availability of a heap space can be guaranteed as the statistics are provided via the V8 GetHeapSpaceStatistics function and may change from one V8 version to the next.

As compared to the GC kind the flags here seem to be even less stable.

This also lacks explanation and they are less self-explanatory compared to the gc kinds. I guess we could link to the v8 docs of `GCCallbackFlags` at least for reference.

Reply:
> This flags provided via V8 [GCCallbackFlags](https://v8docs.nodesource.com/node-13.2/d2/dc3/namespacev8.html#a6748429f1ad36cb4d955e79e4e042229) enum and may change from one V8 version to next.

Looks good for adding after constants list?